test/ocsp/helper/helper.go

```diff
@@ -147,17 +148,27 @@ func Req(fileName string) (*ocsp.Response, error) {
 
 func sendHTTPRequest(req []byte, ocspURL *url.URL) (*http.Response, error) {
 	encodedReq := base64.StdEncoding.EncodeToString(req)
+	var httpRequest *http.Request
+	var err error
 	if *method == "GET" {
 		ocspURL.Path = encodedReq
 		fmt.Printf("Fetching %s\n", ocspURL.String())
-		return http.Get(ocspURL.String())
+		httpRequest, err = http.NewRequest("GET", ocspURL.String(), http.NoBody)
 	} else if *method == "POST" {
 		fmt.Printf("POSTing request, reproduce with: curl -i --data-binary @- %s < <(base64 -d <<<%s)\n",
 			ocspURL, encodedReq)
-		return http.Post(ocspURL.String(), "application/ocsp-request", bytes.NewBuffer(req))
+		httpRequest, err = http.NewRequest("POST", ocspURL.String(), bytes.NewBuffer(req))
 	} else {
 		return nil, fmt.Errorf("invalid method %s, expected GET or POST", *method)
 	}
+	if err != nil {
+		return nil, err
+	}
+	httpRequest.Header.Add("Content-Type", "application/ocsp-request")
```

Comment:
Should this not happen only in the POST branch?

Reply:
(I doubt it breaks anything, just wondering)

Reply:
In theory yes, but then I have to check for err in the POST branch rather than waiting till down here. I figured this was "good enough" and saved a few lines. But also feel free to tell me I am trying to be too clever by putting the err check outside of the if statement. :-)

Reply:
¯\_(ツ)_/¯ it's a test tool, should be fine.